blueprints/model/files/__root__/__path__/__name__.js

```diff
@@ -1,5 +1,5 @@
 <%= importStatements %>
 
 export default Model.extend({
-  <%= attrs %>
+<%= attrs.length ? '  ' + attrs : '' %>
```

Comment:
Ah, ok. Thanks for the heads up @sbatson5. Let's leave it then the way it is, we can always change this later on.